application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -228,11 +234,31 @@ private MessageEmbed makeEmbedForPost(User originalUser, String originalMessage)
     private record ForumPost(User author, Message message) {
     }
 
-    private boolean isBotMessageTransfer(MessageContextInteractionEvent event) {
-        if (event.getTarget().getAuthor().isBot()) {
-            event.reply("Cannot transfer messages from a bot.").setEphemeral(true).queue();
+    private boolean isBotMessageTransfer(User author) {
+        return author.isBot();
+    }
+
+    private void handleBotMessageTransfer(MessageContextInteractionEvent event) {
+        event.reply("Cannot transfer messages from a bot.").setEphemeral(true).queue();
+    }
+
+    private boolean isQuestionTooShort(String question) {
+        return question.length() < INPUT_MIN_LENGTH;
+    }
```

Comment:
do u really need to create a method for these, one liners

Reply:
Originally i started that way but it was a mess inside one single method it's too big already, at least methods abstract away the details to keep it short.